flink-cdc-runtime/src/main/java/com/ververica/cdc/runtime/operators/transform/ColumnTransform.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.runtime.operators.transform;
+
+import com.ververica.cdc.common.types.DataType;
+import org.apache.calcite.sql.SqlBasicCall;
+import org.apache.commons.jexl3.JexlContext;
+import org.apache.commons.jexl3.JexlExpression;
+import org.apache.commons.jexl3.internal.Engine;
+
+import java.io.Serializable;
+
+/** The ColumnTransform applies to describe the information of the transformation column. */
+public class ColumnTransform implements Serializable {
+    private final String columnName;
+    private final DataType dataType;
+    private final SqlBasicCall transform;
+    private final JexlExpression expression;
+    private static final Engine jexlEngine = new Engine();
```

Comment:
I'm a rookie of Calcite and JEXL so please correct me if I'm wrong. It looks like projection and filter expressions are interpreted as SQL syntax first and parsed by Calcite in Projector, then evaluated by JEXL syntax during runtime. Are these two syntaxes compatible to each other?

Reply:
The original intention of using Calcite is to provide parsing capability for Flink function calls. JEXL is a temporary solution that can handle most scenarios, and I will add some unit tests to validate it. Of course, if you have any other good suggestions, we can discuss them together.